scripts/build/ci-deploy/Dockerfile

```diff
@@ -15,30 +15,32 @@ RUN curl -fLO https://storage.googleapis.com/golang/go${GOVERSION}.linux-amd64.t
 
 RUN git clone https://github.com/aptly-dev/aptly $GOPATH/src/github.com/aptly-dev/aptly
 RUN cd $GOPATH/src/github.com/aptly-dev/aptly && \
-    # pin aptly to a specific commit after 1.3.0 that contains gpg2 support
-    git reset --hard a64807efdaf5e380bfa878c71bc88eae10d62be1 && \
     make install
 
 FROM debian:testing-20210111-slim
 
 # Use ARG so as not to persist environment variable in image
 ARG DEBIAN_FRONTEND=noninteractive \
-  GOOGLE_SDK_VERSION=325.0.0 \
-  GOOGLE_SDK_CHECKSUM=374f960c9f384f88b6fc190b268ceac5dcad777301390107af63782bfb5ecbc7
+  GOOGLE_SDK_VERSION=384.0.1 \
+  GOOGLE_SDK_CHECKSUM=437539933ce68aed7147bd03173d53e1c3d69c0694c107bc25e5f8309f29ccb8 \
+  GCSFUSE_VERSION=0.41.1
 
 # Need procps for pkill utility, which is used by the build pipeline tool to restart the GPG agent
 RUN apt update && apt install -yq curl python3-pip procps && pip3 install -U awscli crcmod && \
-    curl -fLO https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-${GOOGLE_SDK_VERSION}-linux-x86_64.tar.gz && \
-    echo "${GOOGLE_SDK_CHECKSUM} google-cloud-sdk-${GOOGLE_SDK_VERSION}-linux-x86_64.tar.gz" | sha256sum --check --status && \
-    tar xzf google-cloud-sdk-${GOOGLE_SDK_VERSION}-linux-x86_64.tar.gz -C /opt && \
-    rm google-cloud-sdk-${GOOGLE_SDK_VERSION}-linux-x86_64.tar.gz && \
+    curl -fLO https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-cli-${GOOGLE_SDK_VERSION}-linux-x86_64.tar.gz && \
+    echo "${GOOGLE_SDK_CHECKSUM} google-cloud-cli-${GOOGLE_SDK_VERSION}-linux-x86_64.tar.gz" | sha256sum --check --status && \
+    tar xzf google-cloud-cli-${GOOGLE_SDK_VERSION}-linux-x86_64.tar.gz -C /opt && \
+    rm google-cloud-cli-${GOOGLE_SDK_VERSION}-linux-x86_64.tar.gz && \
```

Comment:
Is this `s/sdk/cli/` change something that Google has changed, or are we downloading something different here?

Reply:
I checked docs site and for me it looks like they just renamed it.
But for now lets check with less changes and move to newer versions and naming some time later.